presidio_evaluator/evaluation_helpers.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+from typing import List, Dict
+from collections import Counter
+
+from presidio_evaluator import Span
+from presidio_evaluator.evaluation import SpanOutput
+
+
+def get_matched_gold(predicted_span: Span, 
+                    annotated_span: List[Span], 
+                    overlap_threshold) -> SpanOutput:
+        """
+        Given a predicted_span, get the best matchest annotated_span based on the overlap_threshold. 
+        Return a SpanOutput
+        :param sample: InputSample
+        :param pred_span: Span,  Predicted span
+        :param gold_span: List[Span]: List of gold spans from the annotation input
+        """
+        return SpanOutput(output_type="",
+                            predicted_span=None,
+                            annotated_span=None,
+                            overlap_score=0
+                            )
+
+def find_overlap(true_range, pred_range):
+    """Find the overlap between two ranges
+    Find the overlap between two ranges. Return the overlapping values if
+    present, else return an empty set().
+    Examples:
+    >>> find_overlap((1, 2), (2, 3))
+    2
+    >>> find_overlap((1, 2), (3, 4))
+    set()
+    """
+
+    true_set = set(true_range)
+    pred_set = set(pred_range)
+
+    overlaps = true_set.intersection(pred_set)
+
+    return overlaps
+
+def span_compute_actual_possible(results: dict) -> dict:
+    """
+    Takes a result dict that has been output by compute metrics.
+    Returns the results dict with actual, possible populated.
```

Comment:
I would propose to update the doc string further and to explain what's "Actual"  and "possible" refer to.
Add the formulas into the docstring as well